paasta_tools/kubernetes_tools.py

```diff
@@ -1164,8 +1169,42 @@ def get_pod_template_spec(
             ),
         )
 
-    def get_node_selector(self) -> Mapping[str, str]:
-        return {"yelp.com/pool": self.get_pool()}
+    def get_node_affinity(self) -> V1NodeAffinity:
+        """Converts deploy_whitelist and deploy_blacklist in node affinities.
+
+        note: At the time of writing, `kubectl describe` does not show affinities,
+        only selectors. To see affinities, use `kubectl get pod -o json` instead.
```

Comment:
This is another reason I would like to keep node selectors. At least, we can get more info for `kubectl describe`.